code/LINDA/LINDA_fire.dm

```diff
@@ -152,18 +152,18 @@
 // Garbage collect itself by nulling reference to it
 
 /obj/effect/hotspot/proc/Kill()
-	air_master.hotspots -= src
-	DestroyTurf()
-	qdel(src)
+	PlaceInPool(src)
```

Comment:
Um, again, this really isn't intended to be directly called- when a fire runs out of fuel, oxygen, or is extinguished, it calls kill, then it gets Destroy()ed which turns off the lights, you can't just directly pool it.